### PR TITLE
New release v2.0.0

### PR DESCRIPTION
**Pull request by holgerd77**
Ok, release notes for the `v2.0.0` release are ready. 😄

Phew, this was getting chaotic to read the relevant changes out the respective PRs since we had so many discussions ongoing, some PRs closed again unmerged, changes of changes,... 🧐

Please give this some through and broader look. It would be really good if we are able to inform users with some comprise overview of the relevant changes they might be aware of when updating.

Ok, anyhow, cool that we get this out (hopefully 😛) finally. 😎

### CODE REVIEW

**Comment by holgerd77** [comment]
Just made some `npm pack` check, as expected.

File: docs/README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@
 
 **Ƭ BufferLike**: _`Buffer` \| [TransformableToBuffer](interfaces/transformabletobuffer.md) \| [PrefixedHexString](#prefixedhexstring) \| `number`_
 
-_Defined in [types.ts:19](https://github.com/alcuadrado/ethereumjs-tx/blob/84f5b82/src/types.ts#L19)_
```
Comment (alcuadrado) [inline docs/README.md]: I haven't noticed this 😓

Comment (holgerd77) [reply docs/README.md]: Oh, to admit, I haven't noticed the personal repo reference either, wondered on first reading what you refer to. I just re-created the docs for a final run (has turned out to be generally advisable before a release, sometimes it's accurate, sometimes it's not) and didn't put much further thought into it or had some final look. 😄

Comment (alcuadrado) [reply docs/README.md]: Maybe we should only re-create them only before publishing new releases?

Comment (holgerd77) [reply docs/README.md]: Yes, in most cases that makes sense and is normally also the current (relatively 😛) established practice.

**Pull request merged**